features/publishing-a-manual.feature

```diff
@@ -8,7 +8,9 @@ Feature: Publishing a manual
 
   Scenario: Publish a manual
     Given a draft manual exists with some sections
-    When I publish the manual
+    When I click the publish manual button
+    Then I am shown a warning page
```

Comment:
```suggestion
    Then I am asked to confirm the publishing
```

I think "warning page" isn't the right phrasing for this type of page as it is a confirm step. I also think we should be a bit more specific in the Cucumber step as this step isn't for a generic warning (or confirm) page it's strictly for the publishing confirmation